src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Sort all persons in address book by the parameter.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sort all persons by name or appointment date.\n "
+            + "Parameters: KEYWORD [PARAMETER name/ or appt/]\n"
```

Comment:
```suggestion
            + "Parameters: KEYWORD [PARAMETER n/ or appt/]\n"
```

would you think that this change would make the format more streamlined and less convoluted, given the fact that our add command has a prefix of "n/" instead of "name/"?

Reply:
fixed!